src/overmap.cpp

```diff
@@ -2345,7 +2345,7 @@ void overmap::draw(WINDOW *w, WINDOW *wbar, const tripoint &center,
             const auto &ter = ccur_ter.obj();
 
             mvwputch( wbar, 1, 1, ter.get_color(), ter.get_sym() );
-            std::vector<std::string> name = foldstring(ter.get_name(), 25);
+            std::vector<std::string> name = foldstring(_( ter.get_name().c_str() ), 25);
```

Comment:
The `oter_type_t::name` should be translated right after reading (I mean [here](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/overmap.cpp#L412)), not during each utilization.
Furthermore, there's a similar PR in the queue and it was pushed earlier.